.github/workflows/main.yml

```diff
@@ -70,4 +70,4 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           file: ./coverage/lcov.info
-          fail_ci_if_error: true
+          fail_ci_if_error: false
```

Comment:
I've initialized this repo on codecov by simply visiting the magic url: https://app.codecov.io/gh/digitalbazaar/vpqr

Uploads should work properly now, and this can be set to true as with other projects.

Reply:
Oh excellent. So what should our policy be in terms of this `fail_ci_if_error` flag? Seems like `false` is a safe value?

Reply:
true works fine for the most part, which is what *everything* is set to now.  We would like to ensure that coverage is getting uploaded properly and without the `true` flag, it will fail silently.  The main oddity here is that for a new repository, one must initialize the repository on codecov by visiting it's future home using a URL like the one above.